cpp/include/cudf/replace.hpp

```diff
@@ -16,11 +16,70 @@
 
 #pragma once
 
-#include "cudf/cudf.h"
-#include "cudf/types.hpp"
-#include "cudf/column/column.hpp"
+#include <cudf/column/column.hpp>
```

Comment:
```suggestion
#include <cudf/types.hpp>
```
I think all you really need here is `types.hpp`.

Reply:
Changed.